tests/data_manager_list.yaml

```diff
@@ -7,4 +7,6 @@ tools:
   owner: devteam
 - name: data_manager_bwa_mem_index_builder
   owner: devteam
+  revisions:
+    - e519769cdb9c
```

Comment:
Can you try 46066df8813d as well, which is the latest version? I just tested this revision and it works, my hypothesis is that not specifying a revision doesn't install the latest revision after these changes.

Reply:
Yeah sure, will try that.

Reply:
Do you have the log / commit somewhere ?